docs/build/build-with-thirdweb/one-click-deploy.md

```diff
@@ -34,8 +35,10 @@ By the end of this tutorial, you will:
 7. Click **`Deploy Now`** to finalize the process.  
 8. After deployment, well’ll be redirected to the dashboard to upload your NFTs.  
 9. Provide a **name**, upload an **image**, add a **description**, and define **traits** for the NFT.  
-10. **Lazy Mint** the NFT.  
-11. Copy the **`contract address`** from the NFT dashboard.  
+10. **Lazy Mint** the NFT.
+11. Repeat steps 9 and 10 a few times—we need at least **3 NFTs**.  
+12. Copy your **`contract address`** from the NFT dashboard.
+13. Copy the **`contract address`** from the NFT dashboard.  
```

Comment:
I don't understand the difference between copy "your" and copy "the"

Reply:
Oh 😅 Didn't realize it was duplicated...will remove the last one...